Rg.Plugins.Popup/Platforms/Ios/Impl/PopupPlatformIos.cs

```diff
@@ -50,8 +50,9 @@ public Task AddAsync(PopupPage page)
 
             page.DescendantRemoved += HandleChildRemoved;
 
-            if (UIApplication.SharedApplication.KeyWindow.WindowLevel == UIWindowLevel.Normal)
-                UIApplication.SharedApplication.KeyWindow.WindowLevel = -1;
+            var uiWindow = UIApplication.SharedApplication.KeyWindow ?? UIApplication.SharedApplication.Delegate.GetWindow();
```

Comment:
According to this [SO answer](https://stackoverflow.com/a/58031897/1850606), a better solution to get the window might look like this:
```csharp
public static UIWindow GetKeyWindow(this UIApplication application)
{
    if (!UIDevice.CurrentDevice.CheckSystemVersion(13, 0)) 
        return UIApplication.SharedApplication.KeyWindow;
    
    var window = application
        .ConnectedScenes
        .ToArray()
        .OfType<UIWindowScene>()
        .SelectMany(scene => scene.Windows)
        .FirstOrDefault(window => window.IsKeyWindow);

    return window;
}
```

Reply:
I've just realised there are similar code in this very same class

Reply:
Thanks! I've updated the code to reflect that.